apps/cookbook/src/app/examples/item-example/examples/disclosure-animation.ts

```diff
@@ -0,0 +1,54 @@
+import { Component } from '@angular/core';
+import { animate, state, style, transition, trigger } from '@angular/animations';
+
+const config = {
+  selector: 'cookbook-item-example-disclosure-animation',
+  template: `<kirby-card [hasPadding]="true" (click)="isExpanded = !isExpanded">
+  <kirby-card-header [hasPadding]="false">
+    <kirby-item disclosure="arrow-down" [rotateIcon]="isExpanded">
+      <p class="kirby-text-bold">Disclosure animation example</p>
+    </kirby-item>
+  </kirby-card-header>
+
+  <div [@isExpanded]="isExpanded">
+    <p class="kirby-text-large">A card with animated disclosure icon</p>
+    <p> 
+      lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </p>
```

Comment:
Consider semantics and visual levels:
```suggestion
      <h3 class="kirby-text-bold">Disclosure animation example</h3>
    </kirby-item>
  </kirby-card-header>

  <div [@isExpanded]="isExpanded">
    <p class="kirby-text-bold">A card with animated disclosure icon</p>
    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
```